### ISSUE TITLE
Citing `conda-forge`?

### ISSUE THREAD

**Issue by epassaro** [tardis-sn/tardis#1689]
**Is your feature request related to a problem? Please describe.**
We're going to update TARDIS regularly on  `conda-forge` soon. Should we cite them?

**Describe the solution you'd like**
Citation guidelines: https://conda-forge.org/docs/user/introduction.html#how-can-i-give-credit-to-conda-forge

**Describe alternatives you've considered**
Not citing them :joy_cat: 

**Additional context**

**Comment by wkerzendorf** [comment]
yes - please add this to the documentation acknowledgment section.
